app/models/user.rb

```diff
@@ -1,6 +1,14 @@
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  belts = {
```

Comment:
This probably should be a constant instead

Reply:
and the numbers integers